rules/no-inline-assertions.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const {visitIf} = require('enhance-visitors');
+const createAvaRule = require('../create-ava-rule');
+const util = require('../util');
+
+function report({node, context}) {
+	context.report({
+		node,
+		message: 'Assertions should not be called from an inline function.'
```

Comment:
This would be more correct:

```suggestion
		message: 'The test implementation should not be just an assertion in an inline arrow function.'
```

But would be nice to find a way to shorten the above slightly.